components/script/dom/window.rs

```diff
@@ -183,6 +184,8 @@ pub struct Window {
     history: MutNullableJS<History>,
     custom_element_registry: MutNullableJS<CustomElementRegistry>,
     performance: MutNullableJS<Performance>,
+    #[ignore_heap_size_of = "Arc"]
+    paint_time_metrics: Arc<PaintTimeMetrics>,
```

Comment:
My understanding is that we want independent metrics per Window. Specially since these metrics are supposed to be exposed through the PerformanceObserver API at some point, which is exposed in Window.